WordPressLoginFlow/build.gradle

```diff
@@ -33,8 +36,6 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 29
-        versionCode 2
-        versionName "1.1"
```

Comment:
I assume these are gone because now the version source of truth is in the tag during the plugin distribution process?

Reply:
Well, not necessarily tag and.. this is a library, but you're correct :)

Some libraries use this value in the code as `BuildConfig.VERSION_NAME` and that's discouraged. Also, it is [removed in Android Gradle Plugin 4.1](https://issuetracker.google.com/issues/158695880).

Reply:
Thanks for the confirmation.

I guess a better wording for "source of truth is in the tag during the plugin distribution process" would have been "source of truth is the parameter passed to the S3 distribution process done via the plugin" 😄